src/app/routes/utils/fetchPageData/utils/getUrl/index.test.js

```diff
@@ -37,6 +43,12 @@ describe('getUrl', () => {
     );
   });
 
+  it('should return correct url when path ends in *app.app', () => {
```

Comment:
Would we ever have a scenario where the url is `.amp.app`?

Reply:
No we wouldn't. This is because of the regex seen [here](https://github.com/bbc/simorgh/blob/7ccdd1c5ac52ae7b8268356bd0ddf733181fb4c9/src/app/routes/utils/regex/utils/index.js#L6) which is in
turn used to match Optimo URL paths via [this](https://github.com/bbc/simorgh/blob/7ccdd1c5ac52ae7b8268356bd0ddf733181fb4c9/src/app/routes/utils/regex/utils/index.js#L24) regex pattern.